src/main/java/com/vmtapp/enterprise/service/TicketServiceStub.java

```diff
@@ -68,6 +68,11 @@ public TicketServiceStub(ITicketDao ticketDao){
         
     }
 
+    /**
```

Comment:
Added javadocs to the email validation method describing what the regex matches on.